automation_tools/__init__.py

```diff
@@ -353,6 +353,28 @@ def setup_default_capsule(interface=None):
     )
 
 
+def setup_email_notification(smtp=None):
+    """Configures system to handle email notification.
+
+    NOTE: this task needs a 'katello-service restart', so choose wisely when to
+    call it, preferably before another task that restarts the stack, such as
+    the 'setup_default_capsule' task.
+
+    """
+
+    # edit the config files
+    if smtp is not None:
+        run('sed -i -e "s|address.*|address: {0}|" '
+            '/etc/foreman/email.yaml'.format(smtp))
+
+        hostname = run('hostname', quiet=True).strip()
+        if hostname:
+            domain = hostname.split('.', 1)[1]
```

Comment:
I think I want to remove the code that updates the domain attribute as I don't think it buy us anything